### PR TITLE
Spectre (update - round two)

### PR DESCRIPTION
**Pull request by ThatOneGoblin25**
## About the PR
reopen of #1072
- atmos rework
- wiring rework, connected the AME directly to substation because flickering
- decal rework
- rooms reworked
- added atmos piping to artifact chambers
- added beds
- added power monitor console
- added robotics with vends and tiny surgery
- added Genedrobe with a secret wall below it
- added Smile AI pet
- replaced most consoles with tabletop variants
- replaced soda dispenser with booze-o-mat
- removed blast doors over artifact chamber entrance
- removed medical doctor's wall locker
- trimmed the amount of lights
- tweaked price, from 195000 to 185000
- mapped science techfab
- mapped television
- mapped vacuum tiles
- name from NSS Spectre to NR Spectre
- new description

## Why / Balance
Updating to new mapping standards and enhanced the ship's pure focus on science.

## Media
Spectre
Spectre1

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- Tweak: Updated and reworked the Spectre to current mapping guidelines.

### CODE REVIEW

**Comment by ThatOneGoblin25** [comment]
@dvir001 Any clue why the YAML Linter failed?

File: Resources/Textures/_NF/Structures/Machines/computers.rsi/meta.json
Comment (arimah) [inline Resources/Textures/_NF/Structures/Machines/computers.rsi/meta.json:213]: This entire block is duplicated identically right above, which is causing the YAML linter to fail.

Comment (ThatOneGoblin25) [reply Resources/Textures/_NF/Structures/Machines/computers.rsi/meta.json]: Ah woops

**Comment by ThatOneGoblin25** [comment]
We are so back.

**Comment by ThatOneGoblin25** [comment]
@MagnusCrowe We are so back.

**Comment by MagnusCrowe** [comment]
I mentioned this on discord while discussing the original spectre update PR but it may have gotten missed or forgotten.

> 1) I don't see a reason for a specialist ship like the spectre to have a shipyard when its operation does not call for the operation of smaller vessels.  Example: A ship like McCargo calls for a mothership console because it works in tandum with a delivery shuttle.
> A ship like the Empress calls for a mothership console because it's designed to work in tandum with a fleet of smaller sec ships.
> 2)  Spectre isnt designed like a mothership.  Theres no specialized docking ports for a smaller ship, it's just the two normal docking ports on either side. Example: Crescent has a docking arm for smaller ships, Empress has docking bays.   McCargo has a docking port designed just for the McDelivery.

**Comment by ThatOneGoblin25** [comment]
Checkraze said it would be fine if it was 'in themed' with the shuttle in the old PR when it was generic ships.

> Im not personally opposed to the idea of having the mothership console here, the problem I have is that it should be a very narrow, restricted 'themed' list of mini science drone ships, instead of just a generic list of food/general RP ships. The rest of the changes here are fine

If you're still unsatisfied with the mothership console idea proposal I'll remove it (and hopefully not nuke my branch again).

**Comment by MagnusCrowe** [comment]
I don't get why a fully featured research ship would need a smaller ship that does one of the things that the mothership can already do better.  It's almost like you would only buy that smaller ship to transfer the artifact research pad in the event the one on the Spectre blows up...

**Comment by ThatOneGoblin25** [comment]
Hm I guess there's no good, neat solution for me to keep around the mothership console after all. 
There's no RD server so the idea was to use artifact containers to move it back to the spectre and extract the points off it, avoiding crippling explosions or space lube getting everywhere.

**Comment by ThatOneGoblin25** [comment]
Should be good now @MagnusCrowe

**Comment by MagnusCrowe** [comment]
I've been discussing mapping with the other maintainers and checkraze.  We want to start moving away from including drink/food venders on ships that are not focused on hospitality (like bars and restaurants etc.).  Do you have any ideas on what that big middle room would be like if the booze-o-mat and booze dispenser were removed?

Is there a reason for the comm server on this ship?  Normally those are just used on expedition ships so people can communicate over radio on planets.  You can keep it but it looks like it's blocking access to a secret door that hides a medi-drobe... what's going on with that?

**Comment by ThatOneGoblin25** [comment]
If the bar was removed I'd imagine either tables and a small directional glass area with a tech disk terminal, or just expanding the "breakroom" concept and either adding beds, tables, arcade machines and or sofas.

The telecomms was something the original creator added and I didn't see any reason to remove it. It kinda fits in with the tech room look.

The secret door hides a genetics drobe, it's an easter egg I thought to include since we don't have genetics yet and this is a pure science ship. Basically like how the Caduceus has a virology vend but the job role is non-existent. It also only has geneticist clothing stocked.

**Comment by ThatOneGoblin25** [comment]
Perhaps I could add broken machines to the bar instead so it's something the science crew has to repair?

**Comment by MagnusCrowe** [comment]
You can leave the bar how it is currently, we're still discussing this.  I'll probably be harsher on new ships.  Updating the spectre, we can leave it as is for now.

**Review by MagnusCrowe — APPROVED** [review]

**Comment by ThatOneGoblin25** [comment]
@MagnusCrowe I've worked on two sci ships that had full bars, investigator and now spectre. In both cases I merely replaced the soda dispenser for a booze-o-mat. 

The thinking I was following is keep the bar convenient but only pure alcohol so that people get affected by drunk and slight potential to be poisoned. The worse case scenario is people buying water tanks from cargo for drink the whole shift, or abusing sinks to fill up jugs.

**Comment by MagnusCrowe** [comment]
That's good. I'm going to wait until the upstream merge happens before merging this.  You don't need to do anything else before then.

**Comment by ThatOneGoblin25** [comment]
@MagnusCrowe was there something you wanted changed?

**Pull request merged**
